mmv1/products/compute/FirewallPolicyWithRules.yaml

```diff
@@ -0,0 +1,450 @@
+                      applies to connections through any port.
+                      Example inputs include: ["22"], ["80","443"], and
+                      ["12345-12349"].
+        - !ruby/object:Api::Type::String
+          name: 'action'
+          description: |
+            The Action to perform when the client connection triggers the rule. Can currently be either
+            "allow", "deny", "apply_security_profile_group" or "goto_next".
+          required: true
+        - !ruby/object:Api::Type::Enum
+          name: 'direction'
+          description: |
+            The direction in which this rule applies. If unspecified an INGRESS rule is created.
+          values:
+            - :INGRESS
+            - :EGRESS
+        - !ruby/object:Api::Type::Boolean
+          name: 'enableLogging'
+          description: |
+            Denotes whether to enable logging for a particular rule.
+            If logging is enabled, logs will be exported to the
+            configured export destination in Stackdriver.
+          send_empty_value: true
+        - !ruby/object:Api::Type::Array
+          name: 'targetServiceAccounts'
+          description: |
+            A list of service accounts indicating the sets of
+            instances that are applied with this rule.
+          item_type: Api::Type::String
+        - !ruby/object:Api::Type::String
+          name: 'securityProfileGroup'
+          description: |
+            A fully-qualified URL of a SecurityProfile resource instance.
+            Example:
+            https://networksecurity.googleapis.com/v1/projects/{project}/locations/{location}/securityProfileGroups/my-security-profile-group
+            Must be specified if action is 'apply_security_profile_group'.
+        - !ruby/object:Api::Type::Boolean
+          name: 'tlsInspect'
+          description: |
+            Boolean flag indicating if the traffic should be TLS decrypted.
+            It can be set only if action = 'apply_security_profile_group' and cannot be set for other actions.
+        - !ruby/object:Api::Type::Array
+          name: 'targetResources'
+          item_type: Api::Type::String
+          description: |
+            A list of network resource URLs to which this rule applies.
+            This field allows you to control which network's VMs get
+            this rule. If this field is left blank, all VMs
+            within the organization will receive the rule.
+        - !ruby/object:Api::Type::Boolean
+          name: 'disabled'
+          description: |
+            Denotes whether the firewall policy rule is disabled. When set to true,
+            the firewall policy rule is not enforced and traffic behaves as if it did
+            not exist. If this is unspecified, the firewall policy rule will be
+            enabled.
+  - !ruby/object:Api::Type::Array
+    name: 'predefinedRules'
+    description: A list of firewall policy rules. It must include pre-defined rules.
```

Comment:
Description here doesn't make sense, maybe:
A list of pre-defined rules created by the server. These have priority xxxxx-xxxxx

Reply:
right, fixed